plugins/chat/assets/javascripts/discourse/components/chat-channel.hbs

```diff
@@ -68,14 +68,14 @@
   />
 
   {{#if this.pane.selectingMessages}}
-    <ChatSelectionManager
-      @selectedMessageIds={{this.pane.selectedMessageIds}}
-      @chatChannel={{@channel}}
-      @cancelSelecting={{action
-        this.pane.cancelSelecting
-        @channel.selectedMessages
+    {{@channel.isDirectMessageChannel}}
+    {{@channel.canModerate}}
+    <Chat::SelectionManager
+      @enableMove={{and
```

Comment:
So are we thinking it is better generally to hand state things down like this rather than the component being aware of what to do with context? That makes sense because then the component is more dumb, just want to confirm

Reply:
Yes I think it's a better direction...